tech/database/mariadb/about.md

```diff
@@ -10,15 +10,17 @@ description: Transactional SQL database, an enhanced drop-in replacement for MyS
 [MariaDB](https://mariadb.org/en/about/) is a drop-in replacement of MySQL, forked by the community from the latter. To learn more about MariaDB, visit [upstream feature page](https://mariadb.com/kb/en/mariadb/mariadb-vs-mysql-features/), and to see main differences from MySQL, see [compatibility documentation](https://mariadb.com/kb/en/mariadb/mariadb-vs-mysql-compatibility/).
 
 ## What implementations of MySQL we have in Fedora
-[MariaDB](https://apps.fedoraproject.org/packages/mariadb) is preferred MySQL implementation in Fedora. MariaDB can be usually used instead of MySQL as a drop-in replacement in most practical cases and most of the applications will work exactly the same. However, when you need to install Community MySQL, it is available as [community-mysql package](https://apps.fedoraproject.org/packages/community-mysql) in Fedora repositories.
+[MariaDB](https://src.fedoraproject.org/rpms/mariadb) is preferred MySQL implementation in Fedora. MariaDB can be usually used instead of MySQL as a drop-in replacement in most practical cases and most of the applications will work exactly the same. However, when you need to install Community MySQL, it is available as [community-mysql package](https://src.fedoraproject.org/rpms/community-mysql) in Fedora repositories.
 
 Generally, most of the [documentation for MySQL](http://dev.mysql.com/doc/) is valid also for MariaDB.
 
 ## What versions do we have in Fedora
 
-Fedora usually ships only the most recent stable version of MariaDB, which is 10.0. Learn more about this version at [upstream documentation](https://mariadb.com/kb/en/mariadb/what-is-mariadb-100/). The package is called `mariadb` (client tools) and `mariadb-server` (server daemon).
+Fedora usually ships only the most recent stable version of MariaDB, which is 10.2.
+More versions can be found in [COPR repositories](https://copr.fedorainfracloud.org/coprs/mschorm/).
```

Comment:
Maybe it'd be worth mentioning that that's not an official Fedora package and there're no guaranties whatsoever?

Reply:
Good idea